packages/client/components/AssignFacilitatorMenu.tsx

```diff
@@ -0,0 +1,72 @@
+import {AssignFacilitatorMenu_team} from '../__generated__/AssignFacilitatorMenu_team.graphql'
+import React, {useMemo} from 'react'
+import {createFragmentContainer} from 'react-relay'
+import graphql from 'babel-plugin-relay/macro'
+import DropdownMenuLabel from '../components/DropdownMenuLabel'
+import Menu from '../components/Menu'
+import MenuAvatar from '../components/MenuAvatar'
+import MenuItem from '../components/MenuItem'
+import MenuItemLabel from '../components/MenuItemLabel'
+import useAtmosphere from '../hooks/useAtmosphere'
+import {MenuProps} from '../hooks/useMenu'
+import PromoteNewMeetingFacilitatorMutation from '../mutations/PromoteNewMeetingFacilitatorMutation'
+import avatarUser from '../styles/theme/images/avatar-user.svg'
+
+interface Props {
+  menuProps: MenuProps
+  team: AssignFacilitatorMenu_team
+}
+
+const AssignFacilitatorMenu = (props: Props) => {
+  const {menuProps, team} = props
+  const {newMeeting} = team
+  const {facilitatorUserId, id: meetingId} = newMeeting!
+  const {teamMembers} = team || {teamMembers: []}
+  const assignees = useMemo(
+    () => teamMembers
+      .filter((teamMember) => teamMember.userId !== facilitatorUserId)
+      .sort((a,b) => (a.preferredName > b.preferredName) ? 1 : ((b.preferredName > a.preferredName) ? -1 : 0)),
```

Comment:
well now i'm curious. does this have any advantage over the following? 
`sort((a,b) => a.prefferedName < b.preferredName ? -1 : 1)`

Reply:
I copied from SO, and yeah the original looks verbose. I’ll try this minimal version and verify.